src/main/resources/assets/create/lang/pl_pl.json

```diff
@@ -0,0 +1,1157 @@
+
+	"item.create.shadow_steel_pickaxe": "Cienisty stalowy kilof",
+	"item.create.shadow_steel_mattock": "Cienista stalowa graca",
+	"item.create.shadow_steel_sword": "Cienisty stalowy miecz",
+
+	"item.create.rose_quartz_pickaxe": "Złocony kwarcowy kilof",
+	"item.create.rose_quartz_shovel": "Złocona kwarcowa łopata",
+	"item.create.rose_quartz_axe": "Złocona kwarcowa siekiera",
+	"item.create.rose_quartz_sword": "Złocone kwarcowe ostrze",
+
+	"block.create.copper_ore": "Ruda miedzi",
+	"block.create.copper_block": "Blok miedzi",
+	"block.create.copper_shingles": "Miedziane gonty",
+	"block.create.zinc_ore": "Ruda cynku",
+	"block.create.zinc_block": "Blok cynku",
+	"block.create.brass_block": "Blok mosiądzu",
+	
+	"block.create.andesite_casing": "Andezytowa skrzynia",
+	"block.create.brass_casing": "Mosiądzowa skrzynia",
+	"block.create.copper_casing": "Miedziana skrzynia",
+
+	"block.create.cogwheel": "Koło zębate",
+	"block.create.large_cogwheel": "Duże koło zębate",
+	"block.create.turntable": "Obrotnica",
+	"block.create.gearbox": "Skrzynia biegów",
+	"block.create.gearshift": "Przekładnia biegów",
+	"block.create.clutch": "Sprzęgło",
+	"block.create.shaft": "Wał",
+	"block.create.encased_belt": "Obudowana taśma",
+	"block.create.encased_shaft": "Obudowany wał",
+	"block.create.encased_fan": "Obudowany wiatrak",
+	"block.create.adjustable_pulley": "Przekładnia cięgnowa",
+	"block.create.nozzle": "Dysza",
+	"block.create.hand_crank": "Korba",
+	"block.create.cuckoo_clock": "Zegar z kukułką",
+	"block.create.creative_motor": "Silnik elektryczny",
+	"block.create.belt": "Taśma mechaniczna",
+	"block.create.millstone": "Koło młyńskie",
+	"block.create.crushing_wheel": "Koło mielnicze",
+	"block.create.drill": "Wiertło mechaniczne",
+	"block.create.portable_storage_interface": "Przenośny interfejs magazynowy",
+	"block.create.harvester": "Podbieracz mechaniczny",
+	"block.create.saw": "Piła mechaniczna",
+	"block.create.water_wheel": "Koło wodne",
+	"block.create.mechanical_press": "Prasa mechaniczna",
+	"block.create.mechanical_mixer": "Mikser mechaniczny",
+	"block.create.deployer": "Ręka mechaniczna",
+	"block.create.basin": "Misa",
+	"block.create.mechanical_crafter": "Rzemielśnik mechaniczny",
+	"block.create.flywheel": "Koło zamachowe",
+	"block.create.furnace_engine": "Silnik spalinowy",
+	"block.create.speed_gauge": "Prędkościomierz",
+	"block.create.stress_gauge": "Stresomierz",
+	"block.create.cart_assembler": "Dźwig wagonowy",
+	"block.create.analog_lever": "Dźwignia analogiczna",
+	"block.create.rotation_speed_controller": "Kontroler prędkości obrotów",
+
+	"block.create.sticky_mechanical_piston": "Lepki tłok mechaniczny",
+	"block.create.mechanical_piston": "Tłok mechaniczny",
+	"block.create.mechanical_piston_head": "Głowa tłoka mechanicznego",
+	"block.create.piston_pole": "Ramie tłoka mechanicznego",
+	"block.create.mechanical_bearing": "Wiertarka mechaniczna",
+	"block.create.clockwork_bearing": "Wiertarka zegarowa",
```

Comment:
Again as the above, the correct translation would be: _"Łożysko zegarowe"_

Reply:
The same as above.